desktop/use-desktop/images.md

```diff
@@ -145,3 +145,28 @@ To connect a new Artifactory registry to Docker Desktop:
 When signed in, a new **Artifactory** tab appears in the **Images** view.
 By default, the image list shows images sorted by push date: the newest images appear higher in the list.
 
+### GitLab
+
+The GitLab integration lets you interact with images in the
+and GitLab container registry, directly in the **Images** view of Docker Desktop.
+The integration described here connects your local Docker Desktop client with GitLab.
+You can browse, filter, save, and pull images in the GitLab instance you configure.
+
+#### Connect an Artifactory registry
+
+To connect a new GitLab registry to Docker Desktop:
+
+1. Sign in to an GitLab registry using the `docker login` command:
+
+   ```console
+   $ cat ./password.txt | docker login -u <username> --password-stdin <hostname>
+   ```
+
+   - `password.txt`: text file containing your GitLab password.
+   - `username`: your GitLab username.
+   - `hostname`: hostname for your GitLab instance.
+
+2. Open the **Images** view in Docker Desktop.
+
+By default, the image list shows images sorted by push date: the newest images appear higher in the list.
```

Comment:
Okay, but, I tried this locally. I logged into a GitLab container registry. I know that login worked, because I successfully pulled a container and ran it. All from the command line.

```
$ docker login registry.doit.wisc.edu
Authenticating with existing credentials...
Login Succeeded
```

And yet, in my Docker Desktop, the Images view does not reflect the GitLab container registry.

<img width="1050" alt="Screenshot showing Docker Desktop images pane not showing any indication of GitLab" src="https://user-images.githubusercontent.com/952283/233735672-73ca189f-75b0-4d2d-8459-d267fb29f142.png">

<img width="464" alt="Screenshot showing Docker Desktop version number 4.15.0 (93002)" src="https://user-images.githubusercontent.com/952283/233735714-2f6f260d-5787-4062-ba12-80d047b2c789.png">